### PR TITLE
Fixed pull #83, enlisting and vouching changes

### PR DESCRIPTION
**Pull request by DaKATFailsImmenselyAtCoding**
Enlisting/vouching changes have been done to make them more engaging:
-Enlisting can only be done in gang bases.
-Voucher and vouched have to be in the same zone when one wants to vouch the other.

Now I'll copy the things I had in the older pull and fixed to match Magicks' reorganizing:
-With !unpossessfishingrod, !unpossessrod and !unpossess ghosts should be able to unpossess a fleshbag's rod without having to miss a reel or uninhabit.
-Created !github, !pulls and !tutorial (links to "Category:Gameplay" in the wiki) as external link commands.
-Created alts for all Casino games removing "slime". As an example, to play "slimeroulette" now you can simply do "!roulette".
-!parkour is an alt for !jump and there is a small chance of succeeding at jumping.

I'm pretty sure this code is fine, but I'd still appreciate some more experienced coder checking it. Thanks! :)

### CODE REVIEW

**Review by farafrog44 — COMMENTED** [review]
cringus

File: client.py [hunk 1]
```diff
@@ -594,6 +606,9 @@
 	ewcfg.cmd_wiki: ewcmd.wiki,
 	ewcfg.cmd_booru: ewcmd.booru,
 	ewcfg.cmd_bandcamp: ewcmd.bandcamp,
+	ewcfg.cmd_github: ewcmd.github
+	ewcfg.cmd_pulls: ewcmd.pulls
+	ewcfg.cmd_tutorial: ewcmd.tutorial
```
Comment (magicksRampage) [inline client.py:611]: missing commas

File: ew/faction.py [hunk 2]
```diff
@@ -25,6 +25,10 @@ async def vouch(cmd):
 		response = "Vouch for whom?"
 		return await ewutils.send_message(cmd.client, cmd.message.channel, ewutils.formatMessage(cmd.message.author, response))
 
+	if vouchee_data,poi != user_data.poi:
```
Comment (magicksRampage) [inline ew/faction.py:28]: vouchee_data.poi

File: ew/spooky.py [hunk 3]
```diff
@@ -23,6 +23,7 @@
 from .user import EwUser
 from .market import EwMarket
 from .slimeoid import EwSlimeoid
+from .spooky import EwFish
```
Comment (magicksRampage) [inline ew/spooky.py:26]: this import makes no sense

**Review by magicksRampage — CHANGES_REQUESTED** [review]
please test your code before submitting a pull request

**Comment by DaKATFailsImmenselyAtCoding** [comment]
Sorry bout those mistakes and removing mechanics without thinkin about it lol, my bad

**Comment by DaKATFailsImmenselyAtCoding** [comment]
Outdated, will do a new pull

**Pull request closed without merging**